internal/extension/extension.go

```diff
@@ -57,14 +80,69 @@ func (em *ExtensionManager) checkAgentRunning() {
 		logger.Debug("Will use the API")
 		em.isExtensionRunning = false
 	} else {
-		req, _ := http.NewRequest(http.MethodGet, em.helloRoute, nil)
-		if response, err := em.httpClient.Do(req); err == nil && response.StatusCode == 200 {
-			logger.Debug("Will use the Serverless Agent")
-			em.isExtensionRunning = true
-		} else {
-			logger.Debug("Will use the API since the Serverless Agent was detected but the hello route was unreachable")
-			em.isExtensionRunning = false
```

Comment:
I am wondering if you think we should still make this call to `/hello` in the case where universal instrumentation is _not_ enabled?

Reply:
Aha yup, good catch. I made a note to add this back but totally forgot 👍 side note: `/hello` is a confusing route name for its actual usage in the extension 😢